modules/misc/news.nix

```diff
@@ -1132,7 +1132,15 @@ in
         time = "2023-06-17T22:18:22+00:00";
         condition = config.programs.zsh.enable;
         message = ''
-          A new modules is available: 'programs.zsh.antidote'
+          A new module is available: 'programs.zsh.antidote'
```

Comment:
Hmm... this is definitely a typo, but for the sake of `git blame` I don't think it's great to have this change in this PR.